optimum/intel/__init__.py

```diff
@@ -11,3 +11,9 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
+
+from .neural_compressor.config import IncConfig, IncPruningConfig, IncQuantizationConfig
```

Comment:
Here I have to include the `neural_compressor` imports in the root init of the `intel` subpackage to build the docs